recipes/autoconf/all/conanfile.py

```diff
@@ -24,6 +24,12 @@ def source(self):
     def requirements(self):
         self.requires("m4/1.4.18")
 
+    def package_id(self):
+        del self.info.settings.arch
```

Comment:
@madebr, this is an old PR, but I'm wondering why arch is removed from package_id? And why build_type is not added to settings like others tools only recipes?

Reply:
Autoconf is a script-only package. It contains no binaries so build_type does not matter.

If I remember right, arch used to be added to also test autoconf on x86.
But that is not the case anymore.

Reply:
I think `arch` can be removed from the recipe completely.